From 316991af5be3bd1e9fa0cdb4f8792da04da859b7 Mon Sep 17 00:00:00 2001
From: Roman Shaposhnik <rvs@cloudera.com>
Date: Fri, 18 May 2012 13:42:21 -0700
Subject: [PATCH 098/795] MR1: CDH-5919. MR1 'hadoop mrgroups' command not working
 (cherry picked from commit f25d5ec50642220cf1cc39eb389330af26fed214)

(cherry picked from commit 119438528278c229db5f71e352f86f7f2bc10755)
---
 bin/hadoop |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bin/hadoop b/bin/hadoop
index d5abc35..d120d4c 100755
--- a/bin/hadoop
+++ b/bin/hadoop
@@ -280,7 +280,7 @@ elif [ "$COMMAND" = "sampler" ] ; then
 elif [ "$COMMAND" = "dfsgroups" ] ; then
   CLASS=org.apache.hadoop.hdfs.tools.GetGroups
   HADOOP_OPTS="$HADOOP_OPTS $HADOOP_CLIENT_OPTS"
-elif [ "$COMMAND" = "mrgroups" ] ; then
+elif [ "$COMMAND" = "mrgroups" -o "$COMMAND" = "groups" ] ; then
   CLASS=org.apache.hadoop.mapred.tools.GetGroups
   HADOOP_OPTS="$HADOOP_OPTS $HADOOP_CLIENT_OPTS"
 elif [[ "$COMMAND" = -*  ]] ; then
-- 
1.7.0.4

